projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -1166,8 +1172,14 @@ export class IgxTimePickerComponent implements
 
             if (this.mode === InteractionMode.DropDown) {
                 settings = this.overlaySettings || this._dropDownOverlaySettings;
-                if (settings.positionStrategy && settings.positionStrategy.settings) {
+
+                if (this.group && settings.positionStrategy) {
```

Comment:
`settings.positionStrategy` is included in all if clauses. Consider extracting it in a wrapper if clause.